contrib/firebaseio/src/main/java/io/DoFirebaseSetWithPriority.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not  use this file except  in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io;
+
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+
+import com.firebase.client.Firebase;
+
+import com.google.cloud.dataflow.sdk.values.KV;
+import utils.FirebaseAuthenticator;
+
+/**
+ * Uses {@link Firebase#setValue(Object, Object)} to set an object with a priority.
+ * This is a bad pattern, and only used to trigger ChildMovedTest. Users should prefer
```

Comment:
if this is only used in test can you move it to the `test` directory instead of the `main` directory? Then you need not worry that a user will use it.